server/channels/store/sqlstore/terms_of_service_store.go

```diff
@@ -16,10 +17,21 @@ import (
 type SqlTermsOfServiceStore struct {
 	*SqlStore
 	metrics einterfaces.MetricsInterface
+
+	termsOfServiceSelectQuery sq.SelectBuilder
 }
 
 func newSqlTermsOfServiceStore(sqlStore *SqlStore, metrics einterfaces.MetricsInterface) store.TermsOfServiceStore {
-	return SqlTermsOfServiceStore{sqlStore, metrics}
+	s := SqlTermsOfServiceStore{
+		SqlStore: sqlStore,
+		metrics:  metrics,
+	}
+
+	s.termsOfServiceSelectQuery = s.getQueryBuilder().
+		Select("Id", "CreateAt", "UserId", "Text").
+		From("TermsOfService")
+
+	return s
```

Comment:
Any reason for not assigning all the fields inline? There's no error checking to be done after all.

Reply:
Just a stylistic choice to match the other stores -- in some cases, the number of fields makes it feel cleaner to "setup" without that extra level of indentation. But I'm honestly 0/5 on the style, but 2/5 on sticking with the pattern for now.